lib/LedgerSMB/Template.pm

```diff
@@ -120,9 +120,12 @@ Returns a list of format names, any of the following (in order) as applicable:
 
 =item new_UI($request, template => $file, ...)
 
-Wrapper around the constructor that sets the path to 'UI', format to 'HTML',
-the user to C<$request->{_user}>, the locale to C<$request->{_locale}
-and leaves auto-output enabled.
+Wrapper around the constructor that sets the following properties:
+
+    path   => 'UI'
+    format => 'HTML',
+    user   => C<$request->{_user}>
+    locale => C<$request->{_locale}
```

Comment:
Missing a terminating `greater than`?

Reply:
Yes - thanks for spotting. In fact, as that is now an indented code block, the C<> quotes don't work at all there.